Tests/Unit/Controller/RedirectControllerTest.php

```diff
@@ -45,19 +46,17 @@ protected function setUp(): void
         $this->controller = new WebsiteRedirectController();
 
         $this->request = $this->prophesize(Request::class);
-        $this->queryBag = $this->prophesize(ParameterBag::class);
+        $this->queryBag = class_exists(InputBag::class) ? new InputBag() : new ParameterBag();
```

Comment:
when already refctoring this to no mock I would go with not mocking the whole Request object.

```
$this->request = Request::create();
```

then you can easily use:

```
$request->query->set('test', 1);
$request->query->set('my-parameter', 'awesome sulu');
```

This avoids have to bridge between parameterbag and inputbag